system.py

```diff
@@ -159,6 +162,15 @@ def checkTools(cls):
     ret = cls.checkVSDebugTools()
     
     return ret
+  
+  @classmethod
+  def checkPythonVersion(cls):
+
+    python_version_number = str(sys.version_info.major) + '.' + str(sys.version_info.minor) + '.' + str(sys.version_info.micro)
+    if sys.version_info.major is not 2:
+      return ('\033[31m' + 'Google GN compilation requires Python version ' + 
```

Comment:
Don't return string. Return Ture or False. Use Logger.printColorMessage to print message.